draft-song-dns-wireformat-http-01.xml

```diff
@@ -371,6 +398,12 @@
     	<t>Registration for a new HTTP header field: Proxy-DNS-Transport</t>
     
```

Comment:
If this draft is informational type, can we ask IANA to addd a new HTTP header field. or should we go to HTTP related WG to ask for such IANA consideration?